lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -51,10 +51,13 @@ class TurboGraft.Remote
 
   createPayload: (form) ->
     if form
-      if form.querySelectorAll("[type='file']").length > 0
-        formData = new FormData(form)
+      cleanForm = form.cloneNode(true)
+      for node in cleanForm.querySelectorAll('input:not([name])')
```

Comment:
this should be `input:not([name]), textarea:not([name])`

Or alternatively `tg-remote-noserialize` lol